exdb/tests/integration_tests.py

```diff
@@ -406,16 +408,17 @@ def test_does_not_get_drafts_when_not_author(self):
     @override_settings(HALLSTAFF_UPCOMING_TIMEDELTA=timedelta(days=0), RA_UPCOMING_TIMEDELTA=timedelta(days=31))
     def test_week_ahead(self):
         self.create_experience('ad')
-        Experience.objects.get_or_create(author=self.clients['ra'].user_object,
-                                         name="E1", description="test description",
-                                         start_datetime=(now() + timedelta(days=2)),
-                                         end_datetime=(now() + timedelta(days=3)),
-                                         type=self.create_type(),
-                                         subtype=self.create_subtype(),
-                                         goals="Test Goal",
-                                         audience="b",
-                                         status="ad",
-                                         attendance=3)
+        e = Experience.objects.get_or_create(author=self.clients['ra'].user_object,
+                                             name="E1", description="test description",
+                                             start_datetime=(now() + timedelta(days=2)),
+                                             end_datetime=(now() + timedelta(days=3)),
+                                             type=self.create_type(),
+                                             goals="Test Goal",
+                                             audience="b",
+                                             status="ad",
+                                             attendance=3)[0]
+        e.subtypes.add(self.create_subtype())
+        e.save()
```

Comment:
Calling `.save()` is unnecessary if you're only modifying a many-to-many. (And it also results in an extra DB query.)